app/assets/javascripts/analytics/custom-dimensions.js

```diff
@@ -46,6 +46,7 @@
       'political-status': {dimension: 7},
       'analytics:organisations': {dimension: 9},
       'analytics:world-locations': {dimension: 10},
+      'publication-status': {dimension: 12, defaultValue: 'other'},
```

Comment:
The downside to that is that we're sending more data with every GA request on every page for every user.